lib/tools/adb-commands.js

```diff
@@ -210,6 +210,30 @@ methods.isValidClass = function isValidClass (classString) {
   return new RegExp(/^[a-zA-Z0-9./_]+$/).exec(classString);
 };
 
+/**
+ * Fetches the fully qualified name of the main activity for the
+ * given package. It is expected the package is already installed on
+ * the device under test.
+ *
+ * ! This method only works since Android 7 (API level 24)
+ *
+ * @param {string} pkg - The target package identifier
+ * @return {string} Fully qualified name of the main activity
+ * @throws {Error} If there was an error while resolving the activity name
+ */
+methods.resolveMainActivity = async function resolveMainActivity (pkg) {
```

Comment:
What about `resolveLaunchableActivity` or `resolveLauncherActivity`?
Maybe the result is an activity name that has `LAUNCHER` category. Then, 'launchable' or 'launcher' is easier to imagine the result is a launcher.

Reply:
changed